src/sentry/api/bases/organization_events.py

```diff
@@ -26,6 +26,11 @@
     'user_count': {
         'aggregations': [['uniq', 'user', 'user_count']],
     },
+    'latest_event': {
+        'fields': [
+            ['argMax', ['id', 'timestamp'], 'latest_event'],
+        ],
+    },
```

Comment:
@markstory  Heads up, the docs above the `SPECIAL_FIELDS` variable may need to be updated.

Reply:
This code is not long for this world. I'll get this updated in my next set of changes.

Reply:
💀